### PR TITLE
Fix link, add context, fit style guide

### PR DESCRIPTION
**Pull request by NukeManDan**
- lines less than 100 char where possible
- use libp2p doc instead on deprecated link
- add prereq of private network tutorial

### CODE REVIEW

**Comment by NukeManDan** [comment]
needs `prettier` run before merge

**Comment by jimmychu0807** [comment]
@NukeManDan If you feel your work is done, please @ ppl, or add ppl as reviewer on the right sidebar. Otherwise ppl are not notified to review your work (thus not moving forward). I think @kaichaosun would be a right reviewer as he is the original author of this tutorial.

**Comment by NukeManDan** [comment]
Got it, I will for sure request a review on PRs that I think are finalized formally from now on, and leave them as drafts and WIP otherwise.

**Comment by NukeManDan** [comment]
This is a very simple patch an I can rebase #833 to this after

File: docs/tutorials/build-permission-network/index.md [hunk 1]
```diff
@@ -2,24 +2,35 @@
...
+> In a **permissioned** network, only authorized nodes are allowed to perform 
+> specific activities, like validate blocks and propagate transactions.
 
 Before play with this tutorial, we expect that:
 
-* You have completed the [Build a PoE Decentralized Application Tutorial](https://substrate.dev/docs/en/tutorials/build-a-dapp/).
-* You are familiar with the [P2P Networking](https://wiki.polkadot.network/docs/en/maintain-guides-how-to-setup-sentry-node#p2p-networking) in Substrate.
+* You have completed the [Add a Pallet Tutorial](https://substrate.dev/docs/en/tutorials/add-a-pallet/)
+* You have completed the [Start a Private Network Tutorial](https://substrate.dev/docs/en/tutorials/start-a-private-network/)
+* You are **conceptually** comforatble with [LibP2P Networking](https://docs.libp2p.io/introduction/what-is-libp2p/)
```
Comment (kaichaosun) [inline docs/tutorials/build-permission-network/index.md:18]: Is it a overkill to point user to libp2p docs?

**Review by kaichaosun — APPROVED** [review]
Looks good, just leave one concern.

**Pull request closed without merging**